lib/modules/home/components/slide_item.dart

```diff
@@ -2,21 +2,49 @@ part of '../home_module.dart';
 
 class SlideItem extends StatelessWidget {
   final SurveyInfo survey;
+  final bool _isEmptied;
```

Comment:
I think creating `_isEmptied` is redundant, because we have nullable variables in `SurveyInfo`, can use them as empty survey instead